omniledger/service/service.go

```diff
@@ -252,13 +253,17 @@ func (s *Service) AddTransaction(req *AddTxRequest) (*AddTxResponse, error) {
 				if id.Equal(req.SkipchainID) {
 					blocksLeft--
 				}
-				if blocksLeft == 0 {
-					return nil, fmt.Errorf("did not find transaction after %v blocks", req.InclusionWait)
-				}
+			case <-time.After(interval):
```

Comment:
the `tooLong` channel is already a timer. Do we need two timers? Maybe we can reduce `tooLongDur`?

Reply:
Of course, you're right. Didn't see that :(

Reply:
We could reduce `tooLongDur` to twice the expected, optimal time:

`tooLongDur := time.Duration(req.InclusionWait) * interval * 2`

Reply:
Will do so in `ol_simplifying`.